service/geth/geth.go

```diff
@@ -211,6 +211,19 @@ func (gs *gethService) startGeth(stdErr *os.File) error {
 		// "--consoleoutput", "split",
 	}
 
+	switch gs.chainParams.ChainId.String() {
+	case params.AlfajoresChainConfig.ChainID.String():
+		gethArgs = append([]string{"--alfajores"}, gethArgs...)
+	case params.BaklavaChainConfig.ChainID.String():
+		gethArgs = append([]string{"--baklava"}, gethArgs...)
+	case params.MainnetChainConfig.ChainID.String():
+		break
+	default:
+		chainErr := fmt.Errorf("unknown ChainID: %s", gs.chainParams.ChainId.String())
```

Comment:
Would it make sense for the default to add back the `--networkid` flag?  That way it could be used without custom networks as well, if that's desirable.

Reply:
I don't think it's likely/possible to be used that way, but I think that's probably a cleaner way, will change